src/collider/CollisionContact.h

```diff
@@ -14,9 +14,9 @@ struct CollisionContact {
 	double distance; // distance travelled to hit point
 	double timestep;
 	int triIdx;
-	void *userData1, *userData2;
 	int geomFlag;
-	
+	void *userData1, *userData2;
```

Comment:
Now it gives me a warning: `field 'userData2' will be initialized after field 'geomFlag'`